src/wp-includes/comment-template.php

```diff
@@ -2371,7 +2371,7 @@ function comment_form( $args = array(), $post_id = null ) {
 				( $req ? $required_indicator : '' )
 			),
 			sprintf(
-				'<input id="author" name="author" type="text" value="%s" size="30" maxlength="245"%s />',
+				'<input id="author" name="author" type="text" value="%s" size="30"  maxlength="245"%s autocomplete="name" />',
```

Comment:
Small edit: This has an extra space before `maxlength`

Reply:
Fixed.